libfortres/include/fortres/stringref.fmod

```diff
@@ -82,8 +82,8 @@
 
 !_PROC_EXPORT(stringref_of_buf)
         function stringref_of_buf( fstrBuf ) result(strRef)
-          character(len=*),  intent(out) :: fstrBuf

```

Comment:
the ``intent(out)`` prevents this routine to be called with string literals.
i.e. calling ``strBuf( "teststring" )`` would issue a compile error.

Reply:
Okay. I see that it works, but I still wonder, if there is any fortran compiler that might leave `fstrBuf` unassigned/undefined when entering `stringref_of_buf` due to plain `intent(out)`.
What about using `intent(inout)`? In that case, it is clear that the actual value of the parameter `fstrBuf` does matter and it would also be impossible to pass plain temporary values.